pupil-spa/src/app/instructions/instructions.component.html

```diff
@@ -0,0 +1,27 @@
+<header class="page-header">
+  <h1 class="heading-xlarge">Instructions</h1>
+</header>
+
+<div class="hidden">
+  <ul class="text list list-bullet">
+    <li>This check has <strong>30 questions</strong>.</li>
```

Comment:
When we add the question service we will need to make this `30 questions` bit dynamic.  I'd suggest adding a TODO in here to that effect.